app/src/gecko/java/org/mozilla/focus/web/WebViewProvider.java

```diff
@@ -397,7 +419,7 @@ public void onNewSession(GeckoSession session, String uri, GeckoResponse<GeckoSe
         }
 
         private GeckoSession.TrackingProtectionDelegate createTrackingProtectionDelegate() {
-           return new GeckoSession.TrackingProtectionDelegate() {
+            return new GeckoSession.TrackingProtectionDelegate() {
```

Comment:
nit: extra space

Reply:
It was actually one space to the left somehow... I dunno